app/views/shared/_tasklist_sidebar.html.erb

```diff
@@ -0,0 +1,6 @@
+<aside class="govuk-tasklist-sidebar">
```

Comment:
What is this class for?

Reply:
I was using it to scope local styles, however now we're resetting within the component it's a little redundant.  We are using it within the tests to verify the behaviour of the ab test.

Reply:
I'd recommend changing this to something that indicates it's used by smokey to test GOV.UK.

This ensures developers don't use this class to add styling, and properly communicates why it exists.

We do something similar for binding to components with JavaScript '`.js-tasklist` 

for this we could do something like

`.smokey-tasklist-sidebar` or `.qs-tasklist-sidebar` etc

See https://csswizardry.com/2015/03/more-transparent-ui-code-with-namespaces/ for more context

Reply:
That's nice, I'll do that, thanks.